src/draft/Person/JobApplicantProfile.py

```diff
@@ -0,0 +1,4757 @@
+        None,
+        title="Education level",
+        description="The code of the education which the person has acquired. The code levels are based on the "
+        "International Standard Classification of Education (ISCED).",
+        example=EducationLevel.ISCED_LEVEL_7,
+    )
+    education_field: Optional[EducationField] = Field(
+        None,
+        title="Education field",
+        description="The code of education field based on the International Standard "
+        "Classification of Education (ISCED).",
+        example=EducationField.ISCED_FIELD_0731,
+    )
+    graduation_date: Optional[date] = Field(
+        None,
+        title="Graduation date",
+        description="The date on which the specific education has been completed.",
+        example=date(year=2022, month=12, day=8),
+    )
+    education_organization: Optional[str] = Field(
+        None,
+        title="Education organization",
+        description="The name of the organization that provided the education.",
+        example="Institute of Technology",
+        max_length=250,
+    )
+
+
+class LanguageSkill(CamelCaseModel):
+    esco_identifier: Optional[HttpUrl] = Field(
+        None,
+        title="ESCO Identifier",
+        description="The languange skill identifier based on European Standard "
+        "Classification of Occupations (ESCO).",
+        example="http://data.europa.eu/esco/skill/6d3edede-8951-4621-a835-e04323300fa0",
+    )
+    skill_level: CERFLevel = Field(
+        None,
+        title="Skill level",
+        description="The language skill level according to Common European Framework "
+        "of Reference for Languages, known as CERF.",
+        example=CERFLevel.B2,
+    )
+
+
+class OtherSkill(CamelCaseModel):
+    esco_identifier: Optional[HttpUrl] = Field(
+        None,
+        title="ESCO Identifier",
+        description="The skill identifier based on European Standard Classification of "
+        "Occupations (ESCO).",
+        example="http://data.europa.eu/esco/skill/869fc2ce-478f-4420-8766-e1f02cec4fb2",
+    )
+    skill_level: SkillLevel = Field(
+        None,
+        title="Skill level",
+        description="The language skill level according to Common European Framework "
+        "of Reference for Languages, known as CERF.",
+        example=SkillLevel.BEGINNER,
+    )
+
+
+class Certification(CamelCaseModel):
+    certification_name: str = Field(
+        None,
+        title="Certification name",
+        description="The name of the acquired qualification.",
+        example="Hygiene passport",
+        max_length=250,
+    )
+    qualification_type: str = Field(
+        None,
+        title="Qualification type",
+        description="Type of qualification clarifies how the qualification has "
+        "been acquired (e.g. Self-organized / Institutional).",
+        example="Institutional",
+        max_length=250,
+    )
+
+
+class Permit(CamelCaseModel):
```

Comment:
Remove the existing fields in the Permit class, see comments below and add just one field. 

institutionalPermits, see the details at https://docs.google.com/spreadsheets/d/1dxvr56DHZx0OLI2_EqYLtrFRNHLu8l4znvWOi1e0PWM/edit#gid=1554796294